airbyte-webapp/src/views/Connector/ConnectorForm/types.ts

```diff
@@ -14,3 +15,9 @@ export type DestinationConnectorCard = Pick<
   DestinationDefinitionReadWithLatestTag,
   "destinationDefinitionId" | "name" | "icon" | "releaseStage"
 >;
+export type SourceConnectorCard = Pick<
+  SourceDefinitionReadWithLatestTag,
+  "sourceDefinitionId" | "name" | "icon" | "releaseStage"
+>;
+
+export type ConnectorCard = (DestinationConnectorCard | SourceConnectorCard) & { id: string };
```

Comment:
I'm not sure `ConnectorCard` it's the best name for this kind of interface, since we already have the `ConnectorCard` component. 
I guess would be better to rely on the return type of the `useSuggestedConnectors` hook, and name it as is - `SuggestedConnector`. Also, we can omit `destinationDefinitionId` and  `sourceDefinitionId` since we use just `id`(`connectorDefinitionId` in a comment above)
```suggestion
export type SuggestedConnector = (
  | Omit<DestinationConnectorCard, "destinationDefinitionId">
  | Omit<SourceConnectorCard, "sourceDefinitionId">
) & { connectorDefinitionId: string };
```

Reply:
Love it! Thanks for the suggestion!!